tck/src/main/java/com/akkaserverless/javasdk/tck/model/localpersistenceeventing/EventSourcedEntityTwoHandler.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.javasdk.tck.model.localpersistenceeventing;
+
+import com.akkaserverless.javasdk.eventsourcedentity.CommandContext;
+import com.akkaserverless.javasdk.eventsourcedentity.EventSourcedEntityBase;
+import com.akkaserverless.javasdk.impl.EntityExceptions;
+import com.akkaserverless.javasdk.impl.eventsourcedentity.EventSourcedEntityHandler;
+import com.google.protobuf.Any;
+
+/** An event sourced entity handler */
+public class EventSourcedEntityTwoHandler
+    extends EventSourcedEntityHandler<String, EventSourcedEntityTwo> {
+
+  public EventSourcedEntityTwoHandler(EventSourcedEntityTwo entity) {
+    super(entity);
+  }
+
+  @Override
+  public String handleEvent(String state, Object event) {
+    if (event instanceof JsonMessage) {
+      return entity().handle(state, (JsonMessage) event);
+    } else {
+      throw new IllegalArgumentException("Unknown event type [" + event.getClass() + "]");
+    }
+  }
+
+  @Override
+  public EventSourcedEntityBase.Effect<?> handleCommand(
+      String commandName, String state, Any command, CommandContext context) {
+    //    try {
+    switch (commandName) {
+      case "EmitJsonEvent":
+        // FIXME Json
+        //          return entity().emitJsonEvent(state, LocalPersistenceEventing.JsonEvent);
+        throw new IllegalStateException("JSON not implemented yet");
```

Comment:
This will of course fail the tck, but I don't want to tackle json in this PR. Is there a way we can disable specific tck tests?

Reply:
Can disable groups of tests. In this case, ignore the `eventing` tests. Example of doing this in https://github.com/lightbend/akkaserverless-java-sdk/pull/195/commits/27f467f4d76f8b731e40cb80b88631a4f47cb0dc. So in `RunTck` should be able to add something like this to disable these tests for now:

```
.withCommand("-Dakkaserverless.tck.ignore-tests.0=eventing")
```